License2Deploy/rolling_deploy.py

```diff
@@ -249,12 +256,12 @@ def launch_new_instances(self, group_name): # pragma: no cover
       exit(self.exit_error_code)
 
     # step 2: waiting for instances coming up and ready
-    logging.info("Waiting maximum 5 minutes for instances to be ready.")
+    logging.info("Waiting maximum {0} minutes for instances to be ready.".format(self.ready_wait[0] * self.ready_wait[1] / 60))
     self.wait_for_new_instances(new_instance_ids, self.ready_wait[0], self.ready_wait[1]) #Wait for new instances to be up and ready
 
     # step 3: waiting for instance health check to be completed
     try:
-      logging.info("Trying for maximum 5 minutes to health-check all instances.")
+      logging.info("Trying for maximum {0} minutes to health-check all instances.".format(self.health_wait[0] * self.health_wait[1] / 60))
```

Comment:
you do this math several times. Maybe you can create a method that accepts a list that returns the output of this logic.